Detections/SecurityEvent/RDP_RareConnection.yaml

```diff
@@ -28,10 +28,10 @@ query: |
   // use left anti to exclude anything from the previous 14 days that is not rare
   | join kind=leftanti (
   SecurityEvent
-  | where TimeGenerated between (ago(starttime) .. ago(endtime))
+  | where TimeGenerated between (ago(endtime) .. ago(starttime))
```

Comment:
Nice catch, this is a bug, but the bug is in the variable being set above in the let statement.  Change line 21 and 22 to this -
  let starttime = 14d;
  let endtime = 1d;